.travis.yml

```diff
@@ -62,4 +62,5 @@ script:
     - make install.tools install.libseccomp.sudo all runc validate TAGS="apparmor seccomp containers_image_ostree_stub"
     - go test -c -tags "apparmor seccomp `./btrfs_tag.sh` `./libdm_tag.sh` `./ostree_tag.sh` `./selinux_tag.sh`" ./cmd/buildah
     - tmp=`mktemp -d`; mkdir $tmp/root $tmp/runroot; sudo PATH="$PATH" ./buildah.test -test.v -root $tmp/root -runroot $tmp/runroot -storage-driver vfs -signature-policy `pwd`/tests/policy.json
+    - ginkgo -v tests/e2e/.
```

Comment:
Do you know if  'ginkgo' calls "go test" under the covers?  It appears to as I'm seeing a lot of the same (all) params for ginkgo as 'go test'.

Reply:
Yeah, a lot of overlap there.